config/jenkins/greendale/student-pc1-browserhistory.ini

```diff
@@ -0,0 +1,8 @@
+[studentpc1-browserhistory]
+case=extract_and_output
+source=student-pc1.dd
+extract_options=--partitions all --vss_stores all --artifact_filters BrowserHistory
```

Comment:
Maybe add some of the artifact definitions that has been causing some of the edge cases 'WindowsExplorerNamespaceCommonPlaces,WindowsAppInitDLLs,WindowsProductName,OperaHistory,WindowsBITSQueueManagerDatabases,MicrosoftAVQuarantine'

Reply:
Maybe those would be better as unit tests? I'd like to keep this config to just be for BrowserHistory, since we care especially about that artifact.